_search-plugins/search-relevance/index.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: default
+title: Search relevance
+nav_order: 55
+has_children: false
+has_toc: false
+redirect_from:
+  - /search-plugins/search-relevance/
+---
+
+# Compare search results
+
+Compare search results is an experimental feature. For updates on the progress of Compare search results and other search relevance features, or if you want to leave feedback that could help improve the feature, join the [discussion on the OpenSearch forum](https://forum.opensearch.org/t/feedback-experimental-feature-compare-search-results/11331).    
```

Comment:
Global: It's a little strange for the formal name of the feature to have only the initial word capitalized. I think we should indicate in some way that this is the proper name of the feature (probably by capitalizing each word).

Reply:
@macrakis What is the official title of the feature? We've been calling it Search Comparison UI, Search Comparison Tool, Search Results Comparison Tool (my favorite). I agree with @natebower on the capitalization.